examples/with-firebase-hosting/package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "next src/app",
     "serve":
       "echo \"for details see:\n\thttps://github.com/firebase/firebase-tools/issues/535 \n\thttps://github.com/zeit/next.js/issues/3167\";",
-    "deploy": "firebase deploy",
+      "predeploy": "mkdir -p \"dist/functions\"",
+      "deploy": "cd \"dist/functions\" && firebase deploy",
```

Comment:
and this shouldn't be required either because https://github.com/zeit/next.js/blob/76582b8e437dadc4dce18ec6ee525acb6f0d2cdb/examples/with-firebase-hosting/firebase.json#L13 should be picked up on `firebase deploy`.

Reply:
I don't know why...but this doesn't work for me...